src/Foundatio/Logging/LoggerExtensions.cs

```diff
@@ -1,16 +0,0 @@
-using System;
-using System.Collections.Generic;
-using Microsoft.Extensions.Logging;
-using Microsoft.Extensions.Logging.Abstractions;
-
-namespace Foundatio.Logging {
-    public static class LoggerExtensions {
-        public static ILogger GetLogger(this object target) {
-            return target is IHaveLogger accessor ? accessor.Logger : NullLogger.Instance;
-        }
-
```

Comment:
why remove BeginScope extension?  helps add a dictionary.

Reply:
Because we are removing logging. Don’t think we should be adding random extensions to libs. Should stay focused on the services we are trying to provide. I actually created a bunch more extension methods for logging. Maybe we should create a separate lib.